src/pymor/algorithms/to_matrix.py

```diff
@@ -0,0 +1,117 @@
+# This file is part of the pyMOR project (http://www.pymor.org).
+# Copyright 2013-2016 pyMOR developers and contributors. All rights reserved.
+# License: BSD 2-Clause License (http://opensource.org/licenses/BSD-2-Clause)
+
+from __future__ import absolute_import, division, print_function
+
+import numpy as np
+import scipy.linalg as spla
+import scipy.sparse as sps
+import scipy.sparse.linalg as spsla
+
+from pymor.operators.block import BlockOperator
+from pymor.operators.constructions import (AdjointOperator, ComponentProjection, Concatenation, IdentityOperator,
+                                           LincombOperator, VectorArrayOperator, ZeroOperator)
+from pymor.operators.numpy import NumpyMatrixOperator
+
+
+def to_matrix(op, format=None, mu=None):
+    """Transfrom construction of NumpyMatrixOperators to NumPy or SciPy array
+
+    Parameters
+    ----------
+    op
+        Operator.
+    format
+        Format of the resulting |SciPy sparray|.
```

Comment:
Do you really mean `sparray`? Never heard of that before. Also, `|foo|` is a subtitution in reST (http://www.sphinx-doc.org/en/stable/rest.html#substitutions). We define all our substitions in `docs/source/substitutions.py`.